doc/ale.txt

```diff
@@ -1170,6 +1170,8 @@ g:ale_list_window_size                                 *g:ale_list_window_size*
   opened automatically for ALE problems. The default of `10` matches the Vim
   default height.
 
+  To have the to have the window height always match the number of error lines, use `-1`. There is no maximum. Only works with location list (not quickfix).
```

Comment:
The line is too long here, you should wrap it to the maximum column size, and you can remove the not about it not working with quickfix after it does.